ballista/client/src/columnar_batch.rs

```diff
@@ -147,10 +147,10 @@ impl ColumnarValue {
         }
     }
 
-    pub fn to_arrow(&self) -> ArrayRef {
+    pub fn to_arrow(&self) -> Result<ArrayRef> {
         match self {
-            ColumnarValue::Columnar(array) => array.clone(),
-            ColumnarValue::Scalar(value, n) => value.to_array_of_size(*n),
+            ColumnarValue::Columnar(array) => Ok(array.clone()),
```

Comment:
What's up with these changes?

Reply:
Good catch, these changes come from Daniel's [DF 33 update](https://github.com/coralogix/arrow-ballista/pull/142).

Reply:
I'll remove them from this PR